Workflow (#2761)
umbraco/UmbracoDocs: Umbraco-Uno/Working-with-Forms/Workflow/index.md

:::

## How do you set up a workflow

In this part, we will learn how to enter the configuration menu for workflows.
We will be working with a form that contains the following:

* A Name field using a short answer
* An Email field using a short answer
* A Question using a long answer

This is how you get into the configuration of your workflow:

1. Navigate to the Forms section of the backoffice.
2. Here, select the form you want to add a workflow to.
3. At the bottom of the page, you will find a section containing a checkmark saying ***on submit***
4. Click ***Configure workflow***
5. The pop op window will contain the workflow that currently exists
6. Select the one that says *Sending template email to*

### Editing the existing workflow

If you choose an ***empty form*** when setting up a new form, it will come with a Working workflow. However, you will have to make a slight change to it before it functions optimally.
This can be done in the configuration menu for workflows.

1. Click the box ***Send template email to***

> 2. In the field called ***Sender email***, you should add something like Noreply@something.something.

Reviewer: ```suggestion
2. In the field called ***Sender email***, you should add something like Noreply@something.something
```

3. It is recommended to choose ***Igloo template***

> 4. Hit submit and then hit submit in the next menu as well

Reviewer: Would be good with a quick ending to this.

### Creating a custom workflow

In this section, you will learn how to make a custom workflow that sends an email to your inbox of choice. 

1. First, give the workflow the name you want it to have in the ***Workflow Name*** field.
2. Put a checkmark in the ***Active**** so that the workflow is active.

> 3. The ***Include Sensitive Data*** will be left unchecked.

Reviewer: ```suggestion
3. The ***Include Sensitive Data*** will be left unchecked unless your form contains sensitive data.
```

4. Ensure that the ***Email*** is set to the email which should receive replies from forms.

> 5. Next up is the ***Sender Email*** here. You can add something like Noreply@something.com.

Reviewer: ```suggestion
5. Next up is the ***Sender Email***. Here you can add something like Noreply@something.com.
```

6. If you want to reply directly to the customer from the forms reply, you can add {email} to the ***ReplyToEmail*** this will fetch the email from the field called email in the form.
7. Next, you have the option to choose the subject of the email.
8. In the next field, you can pick a template. It is recommended to use ***Igloo-template.cshtml***.

> 9. Finally, leave ***Attachments*** unchecked unless you need the users to send attachments to your inbox.

Reviewer: ```suggestion
9. Finally, leave ***Attachments*** unchecked, unless you have an upload-field in the form, and want to receive the uploaded files through the workflow.
```
